src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1740,6 +1751,21 @@ def aks_update(cmd,     # pylint: disable=too-many-statements,too-many-branches,
     if disable_ahub:
         instance.windows_profile.license_type = 'None'
 
+    if enable_public_fqdn and disable_public_fqdn:
+        raise CLIError(
+            'Cannot specify "--enable-public-fqdn" and "--disable-public-fqdn" at the same time')
```

Comment:
Please use the specific error type `MutuallyExclusiveArgumentError` instead of `CLIError`